lang/python/python-protobuf/Makefile

```diff
@@ -0,0 +1,45 @@
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python-protobuf
+# Must match the version of the `protobuf` package
+PKG_VERSION:=3.17.3
+PKG_RELEASE:=1
+
+PYTHON3_PKG_WHEEL_NAME:=protobuf
+
+PKG_SOURCE:=protobuf-python-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/google/protobuf/releases/download/v$(PKG_VERSION)
+PKG_HASH:=3253c6d17ec0bb6f6382e555cf5ca0a9ffab8d81b691f100f96ce9f5e753018e
+
+PKG_BUILD_DIR:=$(BUILD_DIR)/protobuf-$(PKG_VERSION)
+PYTHON3_PKG_SETUP_DIR:=python
```

Comment:
If you want to use protobuf as suggested wheel_name, thus use it as you should and dont reinvent a wheel here.

It looks like https://pypi.org/project/protobuf/#history and then look at how Python packages are done and use it.